src/components/Company/Offers/Filters/Location.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import TextSearchFilter from "../../../../utils/Table/Filters/TextSearchFilter";
+import { columns } from "../Manage/CompanyOffersManagementSchema";
+
+const Filter = React.forwardRef((props, ref) => (
+    <TextSearchFilter
+        label="Location"
+        placeholder="Search"
+        id="location-filter"
+        column="location"  // column value that will be searched
+        columns={columns}
+        ref={ref}
+        isCaseSensitive={false}
```

Comment:
Following previous discussion, the isCaseSensitive prop should be false as default, then you wouldn't need to pass it here